examples/nextjs-blog/utils/links.js

```diff
@@ -1,27 +1,35 @@
+// eslint-disable-next-line
```

Comment:
What is this for?

Reply:
react import - needed for the svg components otherwise the build fails